code-challenges.test.js

```diff
@@ -13,29 +13,74 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
+
+    describe(`multiplyByThree`, () => {
+        it(`returns an array with all the numbers multiplied by 3`, () => {
+            const numbersArray1 = [6, 7, 8, 9, 10]
+            const numbersArray2 = [24, 27, 30, 33, 36]
+            expect(multiplyByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+            expect(multiplyByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+        })
+    })
+
+// This is a good fail.  ReferenceError: multiplyByThree is not defined
+
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+const multiplyByThree = (arr) => {
+    return arr.map((value) => {
+        return value * 3
+    })
+}
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+
+describe(`divByThree`, () => {
+    it(`decide if the number inside it is evenly divisible by three or not`, () => {
+        const object1 = { number: 15 }
+        const object2 = { number: 0 }
+        const object3 = { number: -7 }
+        expect(divByThree(object1.number)).toEqual(`15 is divisible by three`)
+        expect(divByThree(object2.number)).toEqual(`0 is divisible by three`)
+        expect(divByThree(object3.number)).toEqual(`-7 is not divisible by three`)
+    })
+})
```

Comment:
The prompt states the function should take an object as an argument. In your test, you are passing in a number as an argument instead.